composites/SnippetPreview/components/Button.js

```diff
@@ -0,0 +1,62 @@
+import styled from "styled-components";
+import colors from "../../../style-guide/colors.json";
+
+/**
+ * Returns a basic styled button.
+ *
+ * @param {object} props Component props.
+ *
+ * @returns {ReactElement} Styled button.
+ */
+export const Button = styled.button`
+	display: inline-block;
+	padding: 8px 10px;
+	border: 1px solid ${ colors.$color_button_border };
+	border-radius: 4px;
+	background: ${ colors.$color_button };
+	color: ${ colors.$color_button_text };
+	cursor: pointer;
+	box-sizing: border-box;
+	font-size: inherit;
+	font-family: inherit;
+	font-weight: inherit;
+	outline: none;
+`;
+
+Button.defaultProps = {
+	type: "button",
+};
+
+/**
+ * Returns a button styled for the SnippetPreview.
+ *
+ * @param {object} props Component props.
+ *
+ * @returns {ReactElement} Styled button.
+ */
+export const SnippetPreviewButton = styled( Button )`
+	line-height: 15px;
+	font-size: 0.8rem;
+`;
+
+/**
+ * Returns a component with applied focus styles
+ *
+ * @param {ReactElement} component The original component
+ *
+ * @returns {ReactElement} Component with applied focus styles
+ */
+export function focus( component ) {
```

Comment:
I'd consider to rename this function to better describe what it does. On My Yoast, we're already using this kind of stuff, and the names are a bit more descriptive. For example:
```

disable()
makeButtonFullWidth()
makeResponsiveIconButton()
separatify()
ellipsify()
makeFullWidth()
```